src/TextField/TextField.js

```diff
@@ -23,6 +23,7 @@ export default class TextField extends Component {
      * The CSS class name of the root element.
      */
     className: PropTypes.string,
+    disabled: PropTypes.bool,
```

Comment:
Please could you add a comment for the API docs? It will be a lot of work to go back and add these later. (Trust me, I did it once! :laughing:)